spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarProperties.java

```diff
@@ -887,4 +899,104 @@ public void setParam(Map<String, String> param) {
 
 	}
 
+	public static class Failover {
+
+		/**
+		 * Cluster Failover Policy. Order in which we mentioned in the
+		 * {@link BackupCluster} List
```

Comment:
See [NOTE](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#appendix.configuration-metadata.annotation-processor.automatic-metadata-generation) - only plain text is supported in these config props docs.